src/frontend/screens/MapScreen/GPSPermissions/GPSBackgroundPermissionDisabled.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import {defineMessages, useIntl} from 'react-intl';
+import {StyleSheet, View, Linking} from 'react-native';
+import {Text} from '../../../sharedComponents/Text';
+import {MapPinErrorIconSmall} from '../../../sharedComponents/MapPinErrorIcon/MapPinErrorIconSmall';
+import {Button} from '../../../sharedComponents/Button';
+import * as Location from 'expo-location';
+
+const handleOpenSettings = () => {
+  Linking.sendIntent('android.settings.LOCATION_SOURCE_SETTINGS');
+};
+
+const m = defineMessages({
+  useLocation: {
+    id: 'screens.MapScreen.GPSBackgroundPermissionDisabled.useLocation',
+    defaultMessage: 'Use Your Location',
+  },
+  collectsLocation: {
+    id: 'screens.MapScreen.GPSBackgroundPermissionDisabled.collectsLocation',
+    defaultMessage:
+      'CoMapeo collects location data to track your route on the map, even when the app is closed. Your location data is only stored on your device by default, and is not stored or sent anywhere.',
+  },
+  turnOn: {
+    id: 'screens.MapScreen.GPSBackgroundPermissionDisabled.turnOn',
+    defaultMessage: 'Turn On',
+  },
+});
+
+type GPSBackgroundPermissionDisabledProp = {
+  setBackgroundStatusGranted: (status: boolean) => void;
+};
+
+export const GPSBackgroundPermissionDisabled = ({
+  setBackgroundStatusGranted,
+}: GPSBackgroundPermissionDisabledProp) => {
+  const {formatMessage} = useIntl();
+
+  async function askBackgroundLocationPermission() {
+    const backgroundPermission =
+      await Location.requestBackgroundPermissionsAsync();
+
+    if (backgroundPermission.granted) {
+      setBackgroundStatusGranted(true);
+      return;
+    }
+
+    if (backgroundPermission.canAskAgain) {
+      handleOpenSettings();
+    }
+  }
+  return (
+    <View style={styles.container}>
+      <MapPinErrorIconSmall style={{marginBottom: 20}} />
+      <Text style={styles.title}>{formatMessage(m.useLocation)}</Text>
+      <Text style={styles.description}>
+        {formatMessage(m.collectsLocation)}
+      </Text>
+      <Button fullWidth onPress={askBackgroundLocationPermission}>
+        {formatMessage(m.turnOn)}
+      </Button>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    alignItems: 'center',
+    display: 'flex',
+    justifyContent: 'center',
+    padding: 20,
+    paddingTop: 30,
+  },
+  title: {
+    fontSize: 24,
+    fontWeight: '500',
```

Comment:
looks a bit light compared to the design (and the foreground implementation).

```suggestion
    fontWeight: 'bold',
```

Reply:
I recently learned that we should be following exactly what is on Figma (when it comes to font sizing and weight). The real issue here is that it should be using the font Rubik (as it is a header). We actually are improperly importing Rubik into the app. I have a follow up PR that will fix all the fonts. So I think we should keep the font weight to 500 and when it is converted to rubik it will be consistent with the figma